wire-ios-transport/Source/Public/ZMUpdateEvent+SafeForLogging.swift

```diff
@@ -25,7 +25,7 @@ extension ZMUpdateEvent: SafeForLoggingStringConvertible {
     }
 
     public var safeUUID: String {
-        uuid?.transportString().readableHash ?? "unknown"
+        uuid?.transportString().redactedAndTruncated(maxVisibleCharacters: 7, length: 10) ?? "unknown"
```

Comment:
Should we make a common extension out to `redactedAndTruncated(maxVisibleCharacters: 7, length: 10)`? This is repeated in multiple places and would be challenging to find all for a future change?